src/test_insert_sort.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import copy
+import random
+import pytest
+
+
+TEST_LIST = [
+    [], [8]
```

Comment:
Good edge cases. You should also make sure to include cases with duplicate values. Not as important for this assignment, more so for future sorts.